setup.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='quickcache',
+    version='0.0.1',
```

Comment:
true i'm also a fan of semantic. i more meant changing to 1.0.0. 

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

http://semver.org/spec/v1.0.0.html